readthedocs/core/views/serve.py

```diff
@@ -144,6 +147,7 @@ def _serve_file(request, filename, basepath):
         return response
 
 
+@csrf_exempt
```

Comment:
We should include a comment here about why we're making these exempt (no cookies on doc pages)